### ISSUE TITLE
Single tuple causing cursor.executemany ORA-03147: missing mandatory TTC field

### ISSUE THREAD

**Issue by cjbj** [oracle/python-oracledb#30]
### Discussed in https://github.com/oracle/python-oracledb/discussions/29

<div type='discussions-op-text'>

<sup>Originally posted by **Loncor** June 27, 2022</sup>
I am getting this error when executing the following. Uncommenting the lines below makes it work. So I have concluded if a single tuple in a list is used executemany has a problem. I've got a workaround so not a show stopper. It is not peculiar to this insert as I have tried other tables and differing SQL:-

```
sql_ins_doc_spell = """
            begin
               insert into doc.spell_correct (unknown_word, default_word ) values (:1 , :2);
            exception
                when dup_val_on_index then
                   null;
            end;                              
       """

spell_correct = []

spell_correct.append(('jick', 'jack')) 
#spell_correct.append(('ramana', 'ramona')) 

with connection.cursor() as cursor:
    cursor.executemany(sql_ins_doc_spell, spell_correct)
    connection.commit()
```</div>

**Comment by anthony-tuininga** [comment]
I have pushed a patch to the repository. If you are able to build yourself, let me know if it resolves the problem for you!

**Comment by anthony-tuininga** [comment]
This patch was included in python-oracledb 1.0.2 which was just released.
